File: Daily Challanges/2026-04-April/2026-04-15.py
def sort_and_swap(arr):
    arr.sort()
    for i in range(1, len(arr)):
        if i % 3 == 0:
            temp = arr[i]
            arr[i] = arr[i-1]
            arr[i-1] = temp
    return arr

print(sort_and_swap([3, 1, 2, 4, 6, 5]))

File: Daily Challanges/2026-04-April/2026-04-15.js
function sortAndSwap(arr) {
  arr.sort((a, b) => a - b);
  for (let i = 1; i < arr.length; i++) {
    if (i % 3 == 0) [arr[i], arr[i - 1]] = [arr[i - 1], arr[i]];
  }
  return arr;
}

console.log(sortAndSwap([3, 1, 2, 4, 6, 5]));
